src/main/java/uk/gov/justice/digital/hmpps/keyworker/migration/Movement.kt

```diff
@@ -9,8 +9,8 @@ data class Movement(
   val offenderNo: String,
   val movementDate: LocalDate,
   val movementTime: LocalTime,
-  val fromAgency: String,
-  val toAgency: String,
+  val fromAgency: String?,
+  val toAgency: String?,
```

Comment:
Can't imagine this would ever be null for `ADM` movements but if it was, I guess it would match the `earliestArrivalAtAnotherPrison` check. Null is a different prison though so likely fine anyway!